client/components/auth-form.js

```diff
@@ -11,23 +11,6 @@ const AuthForm = props => {
 
   return (
```

Comment:
when was this form created?

Reply:
This form initially came with the Boiler Plate. Currently it just has email and password. We deleted any other areas of input

Reply:
hi @EvaE-P! this was provided via the boilermaker repo. This is the sign up component. Natalie and I just deleted the "first name" and "last name" input fields that we are not using.